tools/clt_flatten_file_list.cwl

```diff
@@ -0,0 +1,30 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: clt_flatten_file_list
+requirements:
+  - class: InlineJavascriptRequirement
+baseCommand: [echo, done]
+inputs:
+  input_list: { type: 'File[]?' }
+outputs:
+  output:
+    type: File[]?
+    outputBinding:
+      outputEval: |
+        ${
+          var flatten = function flatten(ary) {
+              var ret = [];
+              if (ary != null){
+                for(var i = 0; i < ary.length; i++) {
+                    if(Array.isArray(ary[i])) {
+                        ret = ret.concat(flatten(ary[i]));
```

Comment:
love a recursion

Reply:
Dangerous if we have infinitely nested lists 🐰